src/topology/algebra/infinite_sum.lean

```diff
@@ -757,6 +757,18 @@ begin
   { simp [tsum_eq_zero_of_not_summable hf] }
 end
 
+lemma tsum_ite_eq_add {f : ℕ → ℝ} (hf : summable f) (i : ℕ) (a : ℝ) :
+  ∑' n, ((ite (n = i) a 0) + f n) = a + ∑' n, f n :=
+begin
+  rw [tsum_add ⟨a, has_sum_ite_eq i a⟩ hf, @tsum_eq_single ℝ _ _ _ _ _ i],
+  { simp_rw if_pos rfl },
+  { exact λ j ji, if_neg ji }
+end
```

Comment:
```suggestion
by rw [tsum_add ⟨a, has_sum_ite_eq i a⟩ hf, tsum_ite_eq],
```

Reply:
Or the term-mode proof:
```suggestion
((has_sum_ite_eq _ _).add hf.has_sum).tsum_eq
```

Reply:
I removed the lemma and changed the proof of the one that used it.  Thanks!